utils/http.go

```diff
@@ -19,13 +19,17 @@ package utils
 
 import (
 	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
 	"net"
 	"net/http"
 	"time"
 
 	"golang.org/x/net/http2"
 
 	"github.com/libregraph/lico/version"
+	"github.com/pkg/errors"
```

Comment:
Wrong import group, please move up to the external group.

Reply:
Hm, I'll blame the vim-go plugin here :-) will adjust that.